docs/user-guide/disassembler-backends/binary_ninja.md

```diff
@@ -2,44 +2,60 @@
 
 ## Install
 
-Binary Ninja is not distributed with OFRAK. **You need to have a valid headless BinaryNinja license to build and run the image.** For more details:
-- [Read about the environment setup](../../environment-setup.md#binary-ninja).
-- See the [Docker commands that are run](https://github.com/redballoonsecurity/ofrak/blob/master/disassemblers/ofrak_binary_ninja/Dockerstub).
+Binary Ninja is not distributed with OFRAK. **You need to have a valid Binary Ninja license to use OFRAK Binary Ninja.** You can run OFRAK Binary Ninja natively with a valid **commercial** licence, and in a Docker container with a valid **headless** license.
 
-The recommended BinaryNinja version to use with OFRAK is 3.2.3814. If you are running OFRAK outside of the Docker image, you can switch to this version of BinaryNinja using the [BinaryNinja version switcher](https://github.com/Vector35/binaryninja-api/blob/dev/python/examples/version_switcher.py).
+The recommended Binary Ninja version to use with OFRAK is 3.2.3814. If you are running OFRAK outside of the Docker image, you can switch to this version of Binary Ninja using the [Binary Ninja version switcher](https://github.com/Vector35/binaryninja-api/blob/dev/python/examples/version_switcher.py).
 
+=== "Native"
 
-### Docker
-To build the image, the license should be placed in the project's root directory and named `license.dat`. The serial number needs to be extracted from that file into a file named `serial.txt`. This can be done with the following command:
+    You need to have Binary Ninja installed along with a valid **commercial** Binary Ninja license to run OFRAK Binary Ninja natively. 
 
-```bash
-python3 \
-  -c 'import json, sys; print(json.load(sys.stdin)[0]["serial"])' \
-  < license.dat \
-  > serial.txt
-```
+    1. Create a virtual environment to which you will install code:
+        ```
+        % python3 -m venv venv
+        % source venv/bin/activate
+        ```
+    1. Install `ofrak` and its dependencies.
+    1. Run `make {install, develop}` inside of the ['ofrak_binary_ninja/'](https://github.com/redballoonsecurity/ofrak/tree/master/disassemblers/ofrak_binary_ninja) directory to install OFRAK Binary Ninja.
+    1. Next, install the Binary Ninja Python APIs in your virtual environment
+        ```python
+        % python3 "/Applications/Binary Ninja.app/Contents/Resources/scripts/install_api.py" -v
+        ```
```

Comment:
Everything here being tabbed in is causing it to be interpreted as a literal code block. This should be tabbed back.

Also MkDocs does not like ordered lists with `1.` for all of them. It would be worth building this with MkDocs locally to make sure it builds and looks correct.

Reply:
See above. Make sure you're using the latest changes from master.